spritz/app/src/main/java/com/novoda/spritz/sample/MainActivity.java

```diff
@@ -0,0 +1,122 @@
+import android.widget.Button;
+
+import com.airbnb.lottie.LottieAnimationView;
+import com.novoda.spritz.Spritz;
+import com.novoda.spritz.SpritzPage;
+
+import java.util.concurrent.TimeUnit;
+
+public class MainActivity extends AppCompatActivity {
+
+    public static final int PAGER_NUM = 3;
+    private static final TimeInterpolator SWIPE_FORWARD_INTERPOLATOR = new LinearOutSlowInInterpolator();
+    private static final TimeInterpolator SWIPE_BACKWARDS_INTERPOLATOR = new FastOutSlowInInterpolator();
+
+    private Spritz spritz;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.main);
+
+        LottieAnimationView lottieAnimationView = findViewById(R.id.animation_view);
+        lottieAnimationView.setImageAssetsFolder("images");
+
+        final ViewPager viewPager = findViewById(R.id.viewpager);
+        viewPager.setAdapter(new ScreenSlidePagerAdapter(getSupportFragmentManager()));
+
+        spritz = Spritz
+                .with(lottieAnimationView)
+                .withDefaultSwipeAnimationDuration(300, TimeUnit.MILLISECONDS)
+                .withDefaultSwipeForwardInterpolator(SWIPE_FORWARD_INTERPOLATOR)
+                .withDefaultSwipeBackwardsInterpolator(SWIPE_BACKWARDS_INTERPOLATOR)
+                .withPages(
+                        new SpritzPage.Builder()
+                                .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                                .withSwipeDuration(500, TimeUnit.MILLISECONDS)
+                                .build(),
+                        new SpritzPage.Builder()
+                                .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                                .withSwipeDuration(500, TimeUnit.MILLISECONDS)
+                                .build(),
+                        new SpritzPage.Builder()
+                                .withAutoPlayDuration(500, TimeUnit.MILLISECONDS)
+                                .build()
+                )
+                .attachTo(viewPager);
+
+        Button continueButton = findViewById(R.id.btn_continue);
+        continueButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                int nextItem = viewPager.getCurrentItem() + 1;
+                if (PAGER_NUM > nextItem) {
+                    viewPager.setCurrentItem(nextItem);
+                }
+            }
+        });
+        Button previousButton = findViewById(R.id.btn_previous);
+        previousButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                int prevItem = viewPager.getCurrentItem() - 1;
+                if (prevItem >= 0) {
+                    viewPager.setCurrentItem(prevItem);
+                }
+            }
+        });
+    }
+
+    @Override
+    protected void onResume() {
```

Comment:
This should be in `onStart()`

Reply:
Done in 0a6abe94e0ba611d97570909f42bffbd2187bfe0.